src/components/annotate/css/style.css

```diff
@@ -64,8 +64,8 @@
 
     /*
       When the annotation headings are positioned at the top of the spectrogram
-      (tag-style="spectrogram-top"), if the label is too large to fit inside the
-      annotations width, it can cause the heading to overflow and not be visible.
+      if the label is too large to fit inside the annotations width, it can
+      cause the heading to overflow and not be visible.
```

Comment:
even with the change i'm still confused by this comment.

do you mean:

a) in mode spectrogram-top (then why are we talking about that in .style-edge?)
or b) in mode edge, but when an annotation is rendered right at the top of the spectrogram

Reply:
I'm talking about option b in your example